### PR TITLE
feat: added pan flows component as a json text

### PR DESCRIPTION
**Pull request by WillianLomeu**

### CODE REVIEW

**Review by VMarchena — APPROVED** [review]

**Review by Plezito — APPROVED** [review]

File: src/components/BlipGroupCard.vue [hunk 1]
```diff
@@ -226,11 +221,14 @@ export default {
             status: currentMessage.status,
             reason: currentMessage.reason
           }
+        }
 
-          if (currentMessageMemberInfo) {
```
Comment (Plezito) [inline src/components/BlipGroupCard.vue:230]: Please review this merge and keep this if as was before

File: package-lock.json
Comment (Plezito) [inline package-lock.json:1]: This lockfileVersion is now 3. Did you use node version 14 to install the dependencies?

File: src/components/ApplicationJSon.vue [hunk 2]
```diff
@@ -1,22 +1,25 @@
...
+      :date="date"
+      :on-metadata-edit="isMetadataReady"
+      :on-cancel="cancel"
+      :external-message-text="translations.externalMessageText"
     />
 
-    <menu-list-prompt
-      v-if="document.type === 'interactive' && document.interactive.type === 'list'"
+     <call-to-action
+      v-if="document.type === 'interactive' && document.interactive.type === 'cta_url'"
```
Comment (Plezito) [inline src/components/ApplicationJSon.vue:22]: This should be a v-else-if now that your <json-text> is on top of it

**Review by Plezito — CHANGES_REQUESTED** [review]

**Review by Plezito — APPROVED** [review]

**Review by mpamaro — APPROVED** [review]

**Pull request merged**